### ISSUE TITLE
Can the shopping cart be changed?

### ISSUE THREAD

**Issue by DealsBeam** [arslanstack/affiliate#1]
Can the shopping cart feature be changed to just a "Buy" option with affiliate link that takes you to the affiliate link product page?

**Comment by arslanstack** [comment]
Yes it is possible! feel free to fork and implement changes you want.

**Comment by arslanstack** [comment]
> Yes it is possible! feel free to fork and implement changes you want.
